lms/templates/ux/reference/teams-base.html

```diff
@@ -0,0 +1,165 @@
+        <a href="#" class="dropdown" aria-haspopup="true" aria-expanded="false"><span class="sr">More options dropdown</span> &#9662;</a>
+        <ul class="dropdown-menu" aria-label="More Options" role="menu">
+
+            <li><a href="/account/settings">Account Settings</a></li>
+            <li><a href="/u/frances">My Profile</a></li>
+
+          <li><a href="/logout" role="menuitem">Sign Out</a></li>
+        </ul>
+      </li>
+    </ul>
+  </div>
+</header>
+
+<nav aria-label="Course Material" class="courseware wrapper-course-material">
+  <div class="course-material">
+    <ol class="course-tabs">
+
+          <li>
+             <a class="active" href="/courses/PlantsX/GAR_123/2014_T3/courseware">
+                 Courseware
+                   <span class="sr">, current location</span>
+             </a>
+          </li>
+
+          <li>
+             <a class="" href="/courses/PlantsX/GAR_123/2014_T3/info">
+                 Course Info
+             </a>
+          </li>
+
+          <li>
+             <a class="" href="/courses/PlantsX/GAR_123/2014_T3/discussion/forum">
+                 Discussion
+             </a>
+          </li>
+
+          <li>
+             <a class="" href="/courses/PlantsX/GAR_123/2014_T3/course_wiki">
+                 Wiki
+             </a>
+          </li>
+
+          <li>
+             <a class="" href="/courses/PlantsX/GAR_123/2014_T3/progress">
+                 Progress
+             </a>
+          </li>
+
+          <li>
+             <a class="" href="/courses/PlantsX/GAR_123/2014_T3/instructor">
+                 Instructor
+             </a>
+          </li>
+
+    </ol>
+  </div>
+</nav>
+</%block>
+
+<%block name="headextra">
+<%static:css group='style-course-vendor'/>
+<%static:css group='style-course'/>
+<%block name="nav_skip">${"#seq_content" if section_title else "#course-content"}</%block>
+</%block>
+
+<%block name="js_extra">
+
+  <script type="text/javascript">
+    $(function(){
+        $(".ui-accordion-header a, .ui-accordion-content .subtitle").each(function() {
+          var elemText = $(this).text().replace(/^\s+|\s+$/g,''); // Strip leading and trailing whitespace
+          var wordArray = elemText.split(" ");
+          var finalTitle = "";
+          if (wordArray.length > 0) {
+            for (i=0;i<=wordArray.length-1;i++) {
+              finalTitle += wordArray[i];
+              if (i == (wordArray.length-2)) {
+                finalTitle += "&nbsp;";
+              } else if (i == (wordArray.length-1)) {
+                // Do nothing
+              } else {
+                finalTitle += " ";
+              }
+            }
+          }
+          $(this).html(finalTitle);
+        });
+      });
+  </script>
+
+</%block>
+
+<div class="container">
+  <div class="course-wrapper">
+    <section class="course-content" id="course-content">
+
+    <header class="page-header has-secondary">
+      <div class="header-main">
+        <nav class="breadcrumbs">
```

Comment:
NIT: is it worth including the ``page-header`` parent in mind with the naming of this element's class. Would something like ``page-header-breadcrumbs`` be more descriptive? If we are or will be using the visual ``breadcrumbs`` styling elsewhere, it makes sense to keep it. But if this is for just page heads, let's make it specific.